<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GvQRawDataLoaderDialog</class>
 <widget class="QDialog" name="GvQRawDataLoaderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RAW Data Loader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>3D Model</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Filename</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="_3DModelFilenameLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="_3DModelDirectoryToolButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../GigaVoxelsVoxelizer/Ui/resources.qrc">
          <normaloff>:/icons/Icons/fileopen.png</normaloff>:/icons/Icons/fileopen.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Data Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="_3DModelDataTypeComboBox">
        <item>
         <property name="text">
          <string>USHORT</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>158</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
		 <item row="2" column="0">
			 <widget class="QLabel" name="label_2">
				 <property name="toolTip">
					 <string>Number of levels of resolution of the generated data strucutre.</string>
				 </property>
				 <property name="text">
					 <string>BrickRes</string>
				 </property>
			 </widget>
		 </item>
	    <item row="2" column="1">
			 <widget class="QComboBox" name="_brickSize">
				 <property name="toolTip">
					 <string>Size of the brick</string>
				 </property>
				 <property name="layoutDirection">
					 <enum>Qt::RightToLeft</enum>
				 </property>
				 <property name="currentIndex">
					 <number>2</number>
				 </property>
				 <item>
					 <property name="text">
						 <string>4</string>
					 </property>
				 </item>
				 <item>
					 <property name="text">
						 <string>8</string>
					 </property>
				 </item>
				 <item>
					 <property name="text">
						 <string>16</string>
					 </property>
				 </item>
				 <item>
					 <property name="text">
						 <string>32</string>
					 </property>
				 </item>
				 <item>
					 <property name="text">
						 <string>64</string>
					 </property>
				 </item>
				 <item>
					 <property name="text">
						 <string>128</string>
					 </property>
				 </item>
				 <item>
					 <property name="text">
						 <string>256</string>
					 </property>
				 </item>
			 </widget>
		 </item>
     <item row="3" column="0">
			 <widget class="QLabel" name="label_2">
				 <property name="toolTip">
					 <string>True res.</string>
				 </property>
				 <property name="text">
					 <string>X/Y/Z</string>
				 </property>
			 </widget>
		 </item>
	    <item row="3" column="1">
			 <widget class="QSpinBox" name="_trueXSpinBox">
        <property name="toolTip">
         <string>Scalar value used when streaming data from Host.
It can be used to optimize the internal data structure by flagging nodes as empty.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
			<double>0</double>
        </property>
        <property name="maximum">
			<double>1000000</double>
        </property>
       </widget>
		 </item>
     <item row="3" column="2">
			 <widget class="QSpinBox" name="_trueYSpinBox">
        <property name="toolTip">
         <string>Scalar value used when streaming data from Host.
It can be used to optimize the internal data structure by flagging nodes as empty.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <double>0</double>
        </property>
        <property name="maximum">
			<double>1000000</double>
        </property>
       </widget>
		 </item>
     <item row="3" column="3">
			 <widget class="QSpinBox" name="_trueZSpinBox">
        <property name="toolTip">
         <string>Scalar value used when streaming data from Host.
It can be used to optimize the internal data structure by flagging nodes as empty.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
			<double>0</double>
        
 </property>
        <property name="maximum">
			<double>1000000</double>
        </property>
       </widget>
		 </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../GigaVoxelsVoxelizer/Ui/resources.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GvQRawDataLoaderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GvQRawDataLoaderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
